### ISSUE TITLE
SUPPORT/QUESTION : Global Ignore words VS Show Require Words

### ISSUE THREAD

**Issue by snow3461** [SiCKRAGETV/sickrage-issues#3032]
#INFOS
SR Version: 	BRANCH: (master) / COMMIT: (78e5416c12423ed6c35c0bac9a091ed3fba93261)
Python Version:	2.7.3 (default, Mar 13 2014, 11:03:55) [GCC 4.7.2]

#PROBLEM

I generally watch my tvshows in english, so I have Ignore words set in *Search Setting* as follow :`german,french,core2hd,dutch,swedish,reenc,MrLss,HEVC,x265`

For a specific show, I want to snatch it in french (as its a french production), so I edited the show with Require words as follow : `french`

I got this error
```
2015-09-30 13:11:13 DEBUG    SEARCHQUEUE-MANUAL-263677 :: [T411] :: Invalid scene release: Les.Revenants.S02E01.PROPER.FRENCH.720p.HDTV.x264-PoLY contains french, ignoring it
```
I thought the show settings (i.e Require/Ignore words) superseded the global ones. Am I wrong? 
If so, how can I snatch this specific show with the word *french* in it?

Thanks for your help !

**Comment by miigotu** [comment]
It currently uses both. I guess this could be considered a feature request, but as I see it as "specific shows required should override global ignore" and that just should be the way it works, I'm gonna mark this as a bug.

**Comment by snow3461** [comment]
Ok, so I was not the only one to see it that way ! Thanks @miigotu.

I guess the only workaround right now is to remove *french* from global settings, search wanted episodes, and put *french* back in global setting once it is snatched, will this work?

**Comment by miigotu** [comment]
Yes, or remove it from global and add it to all of the shows you dont want french in
